pipeline/load/display_metrics.py

```diff
@@ -0,0 +1,19 @@
+from typing import Iterator, List
+
+import prefect
+from prefect import task
+
+from pipeline.transform import compute_internal_matching_metrics
+from pipeline.types import Match, MatchingOutput, User
+
+
+@task
+def display_internal_matching_metrics(
+    users: List[User], ranked_matches: Iterator[Match]
+):
+    logger = prefect.context.get("logger")
+    metrics_output = compute_internal_matching_metrics(users, ranked_matches)
```

Comment:
Ah, I see that you called this inside the display task.

Instead, I want you to call the compute task in the flow code, then pass the MatchingMetrics output class to your display function.